master/internal/api_runs.go

```diff
@@ -43,3 +56,178 @@ func (a *apiServer) RunPrepareForReporting(
 		StorageId: storageID,
 	}, nil
 }
+
+func (a *apiServer) SearchRuns(
+	ctx context.Context, req *apiv1.SearchRunsRequest,
+) (*apiv1.SearchRunsResponse, error) {
+	resp := &apiv1.SearchRunsResponse{Runs: []*runv1.FlatRun{}}
```

Comment:
nit: declare variables closer to when they are used

Its kinda weird how we have ```curUser``` after these declarations

Reply:
Not sure what you mean here. `resp`, `curUser`, and `query` are all used in the following lines for getting the project and doing the auth check

Reply:
I was following the same order that exists in SearchExperiments. I can move the get user stuff to the top though